docs/commercial/single-sign-on.rst

```diff
@@ -32,6 +27,10 @@ Using an Identity Provider that supports authentication and authorization allows
 In case you want a user to have access to your documentation project under Read the Docs,
 that user just needs to be granted permissions in the VCS repository associated with it.
 
+You can enable this feature in your organization by going to
+"your organization detail's page" > :guilabel:`Settings` > :guilabel:`Authorization`
+and selecting :guilabel:`GitHub, GitLab or Bitbucket` as provider.
```

Comment:
If the option is literally "GitHub, GitLab, or Bitbucket", this looks good. If these are separate options in the dropdown, they should be individual guilabel roles

Reply:
We have only one option in the drop-down for now since it's the same configuration for all of them.